dev-cpp/yaml-cpp/files/yaml-cpp-0.7.0-gtest.patch

```diff
@@ -1,8 +1,17 @@
+From 30fbefe6102da12e8d4d132aa4af2e24bfda9bfb Mon Sep 17 00:00:00 2001
```

Comment:
Why do you need to change this file?

Reply:
They probably had to rebase the CMake work and went for making the upstream patch usable.

Reply:
Earlier version of patch not accounted that there also should be deleted `include_directories(SYSTEM "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.10.0/googletest/include")` which polluted SYSTEM headers path with directory of bundled dependency. So there need to do additional actions to remove `gtest-1.10.0` directory in ebuild. Reworked patch simplifies ebuild logic.

Reply:
As long as this path doesn't leak to the installed files, I don't see the problem.

But I don't mind the change either. However, if you with to change it, I think a new patch is needed, because this one is used by -r1 which is not removed yet.

The reason why the dir was removed in presubmit before is to avoid using that copy even accidentally in case if some future version starts refering it directly in one more place which is not handled by this patch, either in CMakeLists.txt somehow, or by something like `#include "../gtest-1.10.0/...."`

Reply:
typo: presubmit/prepare

Reply:
> However, if you with to change it, I think a new patch is needed, because this one is used by -r1 which is not removed yet.

0.7.0-r1 applies this patch and compiles without problem.